python/doc_generator.py

```diff
@@ -47,7 +54,12 @@ def _get_transform_markdown(transform_yaml: Dict,
         md.h2('Parameters'),
         md.table(transform_yaml['arguments']),
         md.h2('Example'),
+        md.text(DATASET_PREVIEW_TEXT),
+        md.python_code(DATASET_PREVIEW_CODE),
+        md.text(transform_yaml['preview-data']),
+        md.text(TRANSFORMED_DATA_PREVIEW_TEXT),
         md.python_code(transform_yaml['example_code']),
+        md.text(transform_yaml['post-transform-data']),
```

Comment:
So I don't know exactly what you're using to generate the markdown text for the tables, but we have a library installed already to make them for the Transform args. See the function `md.table`. I think it would be very tedious to do that manually.

Reply:
The `MarkdownTableWriter` is interesting, but maybe not needed here - if we get a Pandas DataFrame type, it has a method called `to_markdown()` that makes it exceedingly simple to generate a markdown representation of the data

Reply:
Another option, if we decide that the output of `to_markdown()` is too loud, would be to directly call the library supporting the method: [tabulate](https://pypi.org/project/tabulate/), which is actually a great pretty data printing library that can be called directly on a Pandas DataFrame